run_Wochenende.py

```diff
@@ -1517,7 +1551,7 @@ def main(args, sys_argv):
 
     parser.add_argument(
         "--fastp",
-        help="Use tool fastp instead of fastqc and trimmomatic",
+        help="Use fastp trimmer instead of fastqc and trimmomatic",
```

Comment:
why instead of fastqc?

Reply:
fastp includes a graphical fastqc style output and is faster than fastqc. We don't use it much so far though.